src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable */
+import Calculate from './components/calculator/Calculate.js';
+import classes from './components/calculator/Calculate.module.css';
 
 function App() {
   return (
```

Comment:
- [ ] Kindly refactor this App component to a class component according to the requirements.